src/test/java/io/gingersnapproject/testcontainers/BaseGingersnapResourceLifecycleManager.java

```diff
@@ -90,13 +97,14 @@ private Map<String, String> internalStart() {
             "gingersnap.database.password", database.getPassword()
       ));
 
+      boolean isOracle = Profiles.isProfileActive("oracle");
```

Comment:
To avoid having `Profiles.isProfileActive("oracle");` throughout the test code, would it be simpler if we just add `Profiles.isOracle()`? That way we don't have the "oracle" string littering the codebase.